react/features/conference/components/native/Conference.tsx

```diff
@@ -621,6 +622,12 @@ export default withSafeAreaInsets(connect(_mapStateToProps)(props => {
         return () => setPictureInPictureEnabled(false);
     }, [ isFocused ]);
 
+    useFocusEffect(useCallback(() => {
```

Comment:
Can't this be combined with the effect above?

Reply:
Yes